### PR TITLE
Add status query parameter to redirect rules

### PR DESCRIPTION
**Pull request by shagabutdinov**
Adds docs for this PR: https://github.com/Constructor-io/autocomplete/pull/1234

### CODE REVIEW

File: source/includes/_redirect_rules.md [hunk 1]
```diff
@@ -146,8 +146,7 @@ key | Yes | The index you'd like to to retrieve redirect rules from.
 num_results_per_page | No | The number of rules to return. Defaults to 20.
 page | No | The page of redirect rules to return. Defaults to 1.
 query | No | Only redirect rules whose url attribute matches or that have at least one redirect rule match whose pattern attribute matches will be returned.
-start_time | No | Only redirect rules whose start_time attribute is greater than or equal will be returned.
-end_time | No | Only redirect rules whose end_time attribute is less than or equal will be returned.
+status | No | One of "current" (shows redirect rules that active now), "pending" (shows redirect rules which start_time is greater than current time), and "expired" (shows redirect rules which end_time is lesser than current time).
```
Comment (adetch) [inline source/includes/_redirect_rules.md]: One of "current" (return redirect rules that are currently active), "pending" (return redirect rules that will become active in the future), and "expired" (return redirect rules that will become active in the future).

File: source/includes/_redirect_rules.md [hunk 2]
```diff
@@ -146,8 +146,7 @@ key | Yes | The index you'd like to to retrieve redirect rules from.
 num_results_per_page | No | The number of rules to return. Defaults to 20.
 page | No | The page of redirect rules to return. Defaults to 1.
 query | No | Only redirect rules whose url attribute matches or that have at least one redirect rule match whose pattern attribute matches will be returned.
```
Comment (adetch) [inline source/includes/_redirect_rules.md]: Return redirect rules whose url or match pattern match the provided query.

File: source/includes/_redirect_rules.md (hunk 1 above)
Comment (shagabutdinov) [reply source/includes/_redirect_rules.md]: Did it but changed expired to `return redirect rules that are not active anymore` - let me know if wording should be different.

File: source/includes/_redirect_rules.md (hunk 2 above)
Comment (shagabutdinov) [reply source/includes/_redirect_rules.md]: Did it.

File: source/includes/_redirect_rules.md (hunk 1 above)
Comment (adetch) [reply source/includes/_redirect_rules.md]: good call :)

**Pull request merged**